web/src/main/java/org/hillview/targets/TableTarget.java

```diff
@@ -309,8 +309,49 @@ HistogramSketch getSketch() {
     @HillviewRpc
     public void histogram(RpcRequest request, RpcRequestContext context) {
         HistogramArgs info = request.parseArgs(HistogramArgs.class);
+    }
+
+    static class PrivateHistogramArgs {
```

Comment:
We will probably have to work together to implement a PrivateTableTarget class.
That class will store per column some of these parameters: epsilon, granularity, min. max.
Some of this code will move there.